src/hooks/useProposals.js

```diff
@@ -139,6 +141,26 @@ export function useProposal(proposalId, appAddress) {
   return [proposalWithData, blockHasLoaded, loadingProposal]
 }
 
+export function useInactiveProposalsWithStake() {
+  const { account } = useWallet()
+  const { honeypot } = useAppState()
+
+  const supporter = useSupporterSubscription(honeypot, account)
+
+  if (!supporter || !supporter.stakes) {
+    return []
+  }
+  const inactiveStakes = supporter.stakes.filter(stake => {
+    return (
+      stake.proposal.type !== ProposalTypes.Decision &&
+      stake.proposal.status !== PROPOSAL_STATUS_ACTIVE_STRING &&
```

Comment:
Should we check for Cancelled or Executed instead? As we do for checking the `withdraw` `mode` on `ProposalActions`?